pkg/daemon/ceph/osd/device.go

```diff
@@ -64,16 +63,20 @@ type osdConfig struct {
 	storeName       string
 }
 
-type Device struct {
-	Name   string `json:"name"`
-	NodeID string `json:"nodeId"`
-	Dir    bool   `json:"bool"`
-}
+// unused?
```

Comment:
Is there a reason to leave this if it's unused?

Reply:
you can delete it